feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt

```diff
@@ -113,12 +113,14 @@ fun DashboardView(
             // TODO navigate to amenity reservations screen
         }
         Text(
+            modifier = Modifier.padding(bottom = sizes.minorRegularSize),
             text = stringResource(R.string.happening_today),
             style = LPTypography.titleMedium,
             color = MaterialTheme.colorScheme.tertiary
         )
         LazyRow(
-            verticalAlignment = Alignment.CenterVertically
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.spacedBy(sizes.smallSize)
```

Comment:
![image](https://user-images.githubusercontent.com/42255248/236003841-f8f8ff79-c42c-47b4-b92d-f6bada6fc7fb.png)

Verify that the size of the card is the same as that of the design

Reply:
Solving

Reply:
Comments solved

Reply:
Please update the screenshot

Reply:
solved